test/test_KernelFunctions_opt.jl

```diff
@@ -0,0 +1,31 @@
+using Cloudy.KernelFunctions
+using JET: @test_opt
+
+rtol = 1e-3
+
+# Constant kernel
+# Initialization
+@test_opt ConstantKernelFunction(FT(π))
```

Comment:
Interesting! What does @test_opt do?

Reply:
It looks for type inference at runtime--we should test all new features with it as well.